dashboard/src/app/get-started/template-list/template-list.controller.ts

```diff
@@ -118,31 +118,43 @@ export class TemplateListController {
       this.$log.error(message);
       return;
     }
+
+    const urls = this.devfileRegistryUrl.split(" ");
```

Comment:
`devfileRegistryUrl` is not a single URL anymore. Let's rename it to `devfileRegistryUrls` then?